Sming/Components/ssl/BearSsl/BrConnection.cpp

```diff
@@ -201,6 +198,14 @@ int BrConnection::runUntil(InputBuffer& input, unsigned target)
 				return BR_ERR_IO;
 			}
 
+			if(wlen == 0) {
+				if(target & BR_SSL_SENDAPP) {
+					return len;
```

Comment:
That looks dodgy; we're saying "yes, 'len' bytes were sent" even though they weren't...